usr/sbin/p11sak/p11sak.h

```diff
@@ -50,7 +50,7 @@ const CK_BYTE brainpoolP384r1[] = OCK_BRAINPOOL_P384R1;
 const CK_BYTE brainpoolP384t1[] = OCK_BRAINPOOL_P384T1;
 const CK_BYTE brainpoolP512r1[] = OCK_BRAINPOOL_P512R1;
 const CK_BYTE brainpoolP512t1[] = OCK_BRAINPOOL_P512T1;
-const CK_BYTE prime192[] = OCK_PRIME192V1;
+const CK_BYTE prime192v1[] = OCK_PRIME192V1;
 const CK_BYTE secp224[] = OCK_SECP224R1;
```

Comment:
Maybe also rename secp224 to secp224r1 ?

Reply:
Done